README.md

```diff
@@ -22,7 +22,7 @@ To enable the remote contacts, on your phone's web interface:
 ## Dependencies (e.g. on Debian)
```

Comment:
Should probably also remove the "(e.g. on Debian)" part, but instead add some other note we need python >= 3.6

Reply:
We could also leave the original instructions in place, in case there are more people that prefer to use the packet management of their host os. Otherwise you are right.

Reply:
Yeah, I'm fine with either